### ISSUE TITLE
Can't install RWF0.23 with Minion7.05, hang on t/minion.t

### ISSUE THREAD

**Issue by tadegenban** [jhthorsen/mojo-ioloop-readwritefork#9]
Hi batman,

I recently can't install  mojo-ioloop-readwritefork witn Minion7.05 which hang on t/minion.t, by Minion7.01, i can successfully installed RWF.
by my investigation, the reason is in recent change Minion not set $SIGCHLD="DEFAULT" in Minion::Command::minion::worker. which let RWF line230 in sub _watch_pid goes to the else condition and use EV::child to watch the child process, EV::child seems not work at all. maybe there's some bug in the EV::child combine use with Mojo ? i'm not sure.

this issue happened on linux. my EV version is 4.22

**Comment by tadegenban** [comment]
Create a sample test to demonstrate the issue of EV::child, in this test, program will hang and not print anything to screen.

```
use strict; use warnings; use 5.012;
use EV;

my $w;
my $c;
my $pid = fork;
if ( $pid ) {
    $SIG{CHLD} = 'DEFAULT'; # 'DEFAULT', 'IGNORE', '', sub {} all not work
    $w = EV::child $pid, 0, sub { say 'catch by ev chld'};
    $c = EV::signal 'CHLD', sub { say 'catch by ev sigal'};

} else {
    sleep 2;
    exit;
}


EV::run;
```

**Comment by jhthorsen** [comment]
Thanks for reporting in and helping out with fixing the issue!

This should now be fixed in 0.24.

**Comment by tadegenban** [comment]
Thanks a lot !
